src/MonsterClass.test.js

```diff
@@ -0,0 +1,22 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import App from './App';
+
+
+// test('a monster button should have the classname monster', () => {
+//     render(<App />);
+//     const monsterButtonEl = screen.getByRole('button', { name: /Amazing! The alien zapped the lizard!/i });
+
+//     fireEvent.click(monsterButtonEl);
+
+//     const alienButtonEl = getb
+// })
+
+test('a monster button should have the classname monster', () => {
+  const container = render(<App />);
+  const monsterButtonEl = screen.getByRole('button', { name: /alien/i });
+  fireEvent.click(monsterButtonEl);
+  const alienButtonEl = container.monsterButtonEl.querySelector('#monster');
+  expect(alienButtonEl).toBeInTheDocument();
```

Comment:
If you examine where your test is failing in the CI logs, or in the terminal for your failed test, you will see that there are multiple elements the with the role button and the name alien.  In this instance you would want to `getAllByRole` and then expect the length of that returned array of buttons to be however many buttons are on the screen that match that role described.  I do not know of an easy way to grab an element by its id like you are trying on line 18.